docs/updating/from_4.0/to_4.1.md

```diff
@@ -5,89 +5,161 @@ latest_tag: '4.1.5'
 
 # Update from v4.0.x to v4.1
 
-This update procedure applies if you are using v4.0.latest.
+This update procedure applies if you are using v4.0.0.
 
 Go through the following steps to update to v4.1.
 
-## Update the app to v4.1
+!!! note
 
-First, run the following command:
+    During the update process you can encounter the following error:
+
+    `Failed to create closure from callable: class 'Ibexa\Bundle\Commerce\Eshop\Twig\SilvercommonExtension' does not have a method 'getNavigation'`
+
+    You can ignore this error, it does not require any action on your part.
+
+## Update the app to latest version of v4.0
+
+First, update your application to the latest version of v4.0: v4.0.8.
```

Comment:
Moved variables partially: unfortunately for 2.5 and 3.3 this is technically not possible because we include snippets there that have the variables generalised (see https://github.com/ezsystems/developer-documentation/blob/master/docs/snippets/update/check_out_version.md)